test/functional/mempool_packages.py

```diff
@@ -65,6 +70,14 @@ def run_test(self):
             witnesstx = self.nodes[0].decoderawtransaction(fulltx, True)
             witness_chain.append(witnesstx['hash'])
 
+            wallet_unspent = self.nodes[0].listunspent(minconf=0)
+            this_unspent = next(i for i in wallet_unspent if i['txid'] == txid)
```

Comment:
This `i` shadows the `i` from the loop above.

I think also this could be a bit more efficient by using the address filter of `listunspent` but it's also not super readable so feel free to ignore:
```
            filter_addr = witnesstx['vout'][0]['scriptPubKey']['address']
            this_unspent = self.nodes[0].listunspent(minconf=0, addresses=[filter_addr])[0]
```

Maybe also add a small comment above this block, like "Check that listunspent ancestor{count, size, fees} yield the correct results"